examples/spark-pi/README.md

```diff
@@ -0,0 +1,33 @@
+#Spark-pi example
+The aim of this example is to show how to run a Spark application in cluster mode on MANTL.
+
+##Prerequisites 
+* Install Spark throug the mantl-api:
+```bash
+curl -k -i -L -X POST -d "{\"name\": \"spark\"}" https://admin:password@control-node/api/1/install
+```
+>**Note**: in the previous command, don't forget to substitute *password* and *control-node* with the actual password, 
+and the actual control node IP (or domain name) of your MANTL cluster.
+
+* Install the GlusterFS addon: http://docs.mantl.io/en/latest/components/glusterfs.html
+
+* SSH into one of your nodes, and create a configuration file `/mnt/container-volumes/spark-conf/spark-defaults.conf` that 

```

Comment:
I don't think you could do that using the mantl-api. @ryane Am i right?
I can think 2 approaches to make this silmpler:
1) Create the configuration directory on GlusterFS when the package is installed. This can be done via the mantl-api (I already tested it). Basically you mount the spark-conf directory in marathon.json, and you write the configuration.
2) Using a custom mantl/spark docker image that reads the secret from zookeeper (assuming that is stored there), and creates the configuration within the container. This would be the best option IMO. I can create this image when I have time, then if you feel like you can think to move it into your repository.